core/src/main/kotlin/org/jetbrains/kotlinx/dataframe/impl/api/compareDataFrames.kt

```diff
@@ -0,0 +1,196 @@
+package org.jetbrains.kotlinx.dataframe.impl.api
+
+import org.jetbrains.kotlinx.dataframe.DataFrame
+import org.jetbrains.kotlinx.dataframe.api.DataRowSchema
+import org.jetbrains.kotlinx.dataframe.api.concat
+import org.jetbrains.kotlinx.dataframe.api.dataFrameOf
+import org.jetbrains.kotlinx.dataframe.api.emptyDataFrame
+import org.jetbrains.kotlinx.dataframe.nrow
+
+internal class ComparisonDescription(
+    val rowAtIndex: Int,
+    val of: String,
```

Comment:
if this only contains "dfA" or "dfB" this could better be an enum. Unfortunately dataframes cannot have a title :/ that would have been even better